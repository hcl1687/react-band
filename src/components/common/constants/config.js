export default (config) => {
  return {
    name: 'constants',
    lazy: false
  }
}
