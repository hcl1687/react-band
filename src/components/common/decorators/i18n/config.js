export default () => {
  return {
    name: 'i18n',
    type: 'decorator',
    lazy: false
  }
}
