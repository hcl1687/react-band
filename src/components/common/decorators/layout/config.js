export default () => {
  return {
    name: '@layout',
    type: 'decorator',
    lazy: false,
    component: 'layout'
  }
}
