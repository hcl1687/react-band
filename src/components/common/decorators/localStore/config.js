export default () => {
  return {
    name: 'localStore',
    type: 'decorator',
    lazy: false
  }
}
