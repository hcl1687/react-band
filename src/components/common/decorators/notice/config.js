export default () => {
  return {
    name: '@notice',
    type: 'decorator',
    lazy: false
  }
}
