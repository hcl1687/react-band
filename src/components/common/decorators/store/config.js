export default () => {
  return {
    name: '@store',
    type: 'decorator'
  }
}
