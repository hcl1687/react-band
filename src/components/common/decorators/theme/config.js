export default () => {
  return {
    name: 'theme',
    type: 'decorator',
    lazy: false
  }
}
