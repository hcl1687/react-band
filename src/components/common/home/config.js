export default (config) => {
  return {
    name: 'home',
    route: {
      path: '/',
      exact: true
    },
    lazy: false,
    // auth: {},
    decorators: ['i18n', 'theme', 'layout']
  }
}
