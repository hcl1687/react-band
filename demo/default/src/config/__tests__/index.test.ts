import config from '../index'

describe('config', () => {
  it('should render correctly', () => {
    expect(config).toMatchSnapshot()
  })
})
