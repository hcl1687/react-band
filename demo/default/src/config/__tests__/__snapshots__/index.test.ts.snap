// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config should render correctly 1`] = `
Object {
  "languages": Array [
    "en",
    "zh-CN",
  ],
  "themes": Array [
    "default",
    "darkgray",
  ],
}
`;
